.pipelines/ci/templates/build-powertoys-ci.yml

```diff
@@ -9,7 +9,7 @@ jobs:
   variables:
     BuildConfiguration: ${{ parameters.configuration }}
     BuildPlatform: ${{ parameters.platform }}
-    useAllCPUsWhileBuildingPowerToys: $[ne(variables['BuildPlatform'],'arm64')] # Multi processor build for arm64 is broken in the latest pipelines update
```

Comment:
You'll also want to remove this which limits the build to only two processors: 

https://github.com/microsoft/PowerToys/blob/a8d2aeedf21a92f9137421c3f5c4149970e7e4ee/.pipelines/ci/templates/build-powertoys-ci.yml#L4

https://github.com/microsoft/PowerToys/pull/21930/files

Reply:
knew i forgot something!  However i think the maxCpu flag actually overrides that as /m.  good callout none the less for us to revert

Reply:
You are correct, that's great to know.  It seems odd that "additional arguments" get overridden by first-class arguments. :sigh:

```
"C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\Bin\msbuild.exe"
"C:\a\_work\1\s\PowerToys.sln"
/nologo
/nr:false
/dl:CentralLogger,"C:\a\_work\_tasks\VSBuild_71a9a2d3-a98a-4caa-96ab-affca411ecda\1.212.0\node_modules\azure-pipelines-tasks-msbuildhelpers-v3\tools\Microsoft.TeamFoundation.DistributedTask.MSBuild.Logger.dll";"RootDetailId=c75d70b5-3df2-42ed-be55-b3565c6240dc|SolutionDir=C:\a\_work\1\s|enableOrphanedProjectsLogs=true"*ForwardingLogger,"C:\a\_work\_tasks\VSBuild_71a9a2d3-a98a-4caa-96ab-affca411ecda\1.212.0\node_modules\azure-pipelines-tasks-msbuildhelpers-v3\tools\Microsoft.TeamFoundation.DistributedTask.MSBuild.Logger.dll"
-m:2
/p:platform="x64"
/p:configuration="Release"
/p:VisualStudioVersion="17.0"
/m
/p:_MSDeployUserAgent="VSTS_ea964927-e753-4550-bf48-b63004033439_build_219_0"
```

Reply:
I have a tool for correcting the case of files so that a repo can build on platforms with case-sensitive file systems.  I think I'll add a feature to it to also look at this scenario of keeping major properties like `Configuration` and `Platform` to be consistent casing across the repo.

https://github.com/jeffkl/msbuild-case-fixer/tree/prototype

Reply:
OMG this is money.  the other thing we want to do is insure all projects have ARM64 and x64.  When new stuff gets added, just solid extra coverage

Reply:
On that note I noticed this when poking around:

![image](https://user-images.githubusercontent.com/3347530/201402534-404b800d-09db-4476-a635-2829273c1955.png)

Intentional or accident? There is a lower-case `amd64` platform for MeasureToolUI.

Reply:
honestly, could be we totally missed that.  Great finds

Reply:
also why i want to have all this stuff automated as a sanity check :)